DRAFT_RELEASE_NOTES.md

```diff
@@ -71,8 +71,25 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.10.0
    </parent>
    ```
 
+### Split Data Records for the Spark Pipeline
+If your spark pipeline is using `aissemble-data-records-separate-module` profile for your data records, you must add the `<version>` tag for
+the `jackson-mapper-asl` dependency artifact in the root pom.xml file to enable the build.
+```xml
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
+            <artifactId>jackson-mapper-asl</artifactId>
+  +         <version>${version.jackson.mapper.asl}</version>
```

Comment:
S: We should just update the MDA template for this.

Reply:
Ah, this is one of the changes we can't make via Baton because the project will be in a broken state as soon as they set the build-parent version to 1.10.